README.md

```diff
@@ -1 +1,95 @@
-# code-buddy-backend
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# CODE BUDDY BACKEND
+
+> This project is about creating API end point using Ruby on Rails and deploying the API on heroku. The API has meetings and reservations endpoints. The API is used in our front end React project which is based on an app to book a meeting to learn or for coding session with your buddy. 
+
+
+## The learning objective of this project are
+
+- Create rails app
+- Connect react front end app to rails back end
+- Use API end point to get data
+- Fetch data from the API
+

```

Comment:
Add the Er Diagram, and mention which features are implemented and which features will be implemented in feature

Reply:
Thank you for your feedback. I did it